test/integration/nkilib/experimental/moe/test_bwmm_per_expert_scale.py

```
@@ -0,0 +1,674 @@
+            data["input_tensor"].float().numpy(),  # BF16 → float32
+            data["weights_sim"].float().numpy(),  # BF16 → float32 (sim FP8)
+            data["weight_scales"].float().numpy(),  # already float32
+            data["block_expert_map"].numpy(),
+            nki_output_np,
+            NUM_BLOCKS=NB,
+            BLOCK_SIZE=B,
+            HIDDEN_DIM=H,
+            INTERMEDIATE_DIM=I,
+            NUM_EXPERTS=E,
+        )
+    except Exception as exc:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)